rnd/autogpt_builder/src/components/flow.css

```diff
@@ -11,16 +11,6 @@ code {
     monospace;
 }
 
-button {
-  background-color: #ffffff;
-  color: #000000;
-  padding: 10px;
-  border: none;
-  border-radius: 4px;
-  cursor: pointer;
-  transition: background-color 0.3s ease;
-}
-
 button:hover {
```

Comment:
We should be able to remove this class as well now. We would want to leverage the variants in our Button from ShadCn for these effects.